models/public/wavernn/description/wavernn.md

```diff
@@ -88,18 +89,20 @@ The wavernn-rnn model accepts two feature maps from wavernn-upsampler and produc
 | MParams                         | 3.83                                      |
 
 ### Input
-1. Time slice in `upsampled_mels`, name: `m_t`. Shape: [Bx80]
-2. Time/space slices in `aux`, name: `a1_t`, `a2_t`, `a3_t`,`a4_t`. Shape: [Bx32]. Second dimension is 32 = aux.shape[1] / 4
-3. Hidden states for GRU layers in autoregression, name `h1.1`, `h2.1`. Shape: [Bx512].
-4. Previous prediction for autoregression (initially equal to zero), name: `x`. Shape: [Bx1]
 
-Note: B - batch size.
+1. Time slice in `upsampled_mels`, name: `m_t`. Shape: `B, 80`
+2. Time/space slices in `aux`, name: `a1_t`, `a2_t`, `a3_t`,`a4_t`. Shape: `B, 32`. Second dimension is 32 = aux.shape[1] / 4
```

Comment:
That should be splitted into separate list items. Also applicable to `h1`, `h2` output layers. Otherwise, it seems that model has 4 input layers.

Reply:
Done